spot_driver/CMakeLists.txt

```diff
@@ -1,7 +1,5 @@
 cmake_minimum_required(VERSION 3.22)
 
-list(APPEND CMAKE_PREFIX_PATH /opt/spot-cpp-sdk)
-
```

Comment:
this line is also in the spot_ros2_control cmake and probably could also be removed if it's not needed anymore

Reply:
Done in https://github.com/bdaiinstitute/spot_ros2/pull/495/commits/93d3b304feac6f10fa26013c81111c4f6483acb5.